graphql_compiler/tests/test_testing_invariants.py

```diff
@@ -7,41 +7,48 @@
 
 
 def get_function_names_from_module(module):
-    return [
+    """Return a set of function names present in a given module."""
+    return {
         member
-        for (member, member_type) in getmembers(module)
+        for member, member_type in getmembers(module)
         if isfunction(member_type)
-    ]
+    }
 
 
-def get_function_names_from_class(test_class):
+def get_test_function_names_from_class(test_class):
+    """Return a set of test function names present in a given TestCase class."""
+    if not issubclass(test_class, unittest.TestCase):
+        raise AssertionError(u'Received non-test class {} as input.'
+                             .format(test_class))
     member_dict = test_class.__dict__
-    return set(
+    return {
         member
         for member in member_dict
         if isfunction(member_dict[member]) and member[:5] == 'test_'
-    )
+    }
 
 
+# The namedtuple function is imported from test_input_data,
+# but does not correspond to any test inputs.
 IGNORED_FUNCTIONS = ['namedtuple']
```

Comment:
make this be `frozenset({'namedtuple'})`

Reply:
you are checking for membership with `in` -- things checked with `in` should be set-like